### PR TITLE
[MBL-17271][Student] Add reminder section

### PR DESCRIPTION
**Pull request by kristofnemere**
Test plan: the new reminder section should appear when there's a future due date. The adding part is not finished, will be covered in the next ticket, but you can add and remove dummy reminders.

refs: MBL-17271
affects: Student
release note: none

## Checklist

- [ ] Tested in dark mode
- [ ] Tested in light mode
- [ ] A11y checked

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kdeakinstructure — APPROVED** [review]
QA 👍

File: apps/student/src/main/java/com/instructure/student/features/assignments/details/AssignmentDetailsViewModel.kt [hunk 1]
```diff
@@ -112,6 +114,15 @@ class AssignmentDetailsViewModel @Inject constructor(
 
     private val submissionQuery = database.submissionQueries.getSubmissionsByAssignmentId(assignmentId, apiPrefs.user?.id.orDefault())
 
+    private val remindersLiveData = assignmentDetailsRepository.getRemindersByAssignmentIdLiveData(
+        apiPrefs.user?.id.orDefault(), assignmentId
+    ).apply {
+        observeForever {
```
Comment (tamaskozmer) [inline apps/student/src/main/java/com/instructure/student/features/assignments/details/AssignmentDetailsViewModel.kt:120]: I think we should clear this observer or use one that is lifecycle aware.

**Review by tamaskozmer — CHANGES_REQUESTED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by tamaskozmer — APPROVED** [review]

**Pull request merged**